mappings/net/minecraft/value/JsonUtils.mapping

```diff
@@ -3,7 +3,7 @@ CLASS ol net/minecraft/value/JsonUtils
 	METHOD a createArray (Ljava/lang/Iterable;)Lqe;
 	METHOD a createString (Ljava/lang/String;)Lqa;
 	METHOD a createObject (Ljava/util/Map;)Lqe;
-	METHOD a createArray ([Lgu;)Lqe;
+	METHOD a aJsonArray ([Lgu;)Lqe;
```

Comment:
is it really called "aJsonArray"

Reply:
yes, other methods are also similarly named but i havent renamed them yet

Reply:
or maybe its not? thoguht it was

Reply:
wait yes it was i am dumb ignore that.